01-managing-files-with-ganga.md

```diff
@@ -246,8 +246,8 @@ j2.outputfiles.get(DiracFile)[0].get(localPath='.')
 ```
 
 > ## Using Ganga v600r44 {.callout}
-> Using `DiracFile` with the `Local` backend doesn't work with Ganga v601r19.
-> You can try again with Ganga v600r44, that happens to work in this instance, 
+> Using `DiracFile` with the `Local` backend has had issues in recent versions.
+> If this does not work, you can try again with Ganga v600r44, that happens to work in this instance, 
```

Comment:
Agreed. I haven't had to resort to 6.0.44 for a while, so I'd said the advice is harmful now.